doc/vips.md

```diff
@@ -123,6 +123,10 @@ Any additional options are forwarded to [`Vips::Image#thumbnail_image`]:
 pipeline.resize_to_fill!(400, 400, crop: :attention) # smart crop
 ```
 
+Crop option is `:centre` by default. Acceptable values are `:none`, `:attention`, `:centre`, `:entropy`, `:attention` and `:last`.
```

Comment:
I'd remove :last. It's a convention in the C API to have an extra enum at the end (called `last`) so you can get the highest value at compile time.